tests/test_client.py

```diff
@@ -585,8 +579,8 @@ def test_get_estimates_no_subunits_reporting(model_client, va_governor_county_da
     data = data_handler.get_percent_fully_reported(0)
 
     preprocessed_data = va_governor_county_data.copy()
-    preprocessed_data["last_election_results_turnout"] = preprocessed_data["baseline_turnout"].copy()
-    preprocessed_data["total_voters_turnout"] = preprocessed_data["last_election_results_turnout"] + 1
+    preprocessed_data["last_election_results_turnout"] = preprocessed_data["baseline_turnout"].copy() + 1
+    #   preprocessed_data["last_election_results_turnout"] = preprocessed_data["last_election_results_turnout"] + 1
```

Comment:
do you mind just removing this line to avoid confusion

Reply:
done